crates/python/src/lib.rs

```diff
@@ -43,5 +43,6 @@ create_init_submodule! {
 
 #[pymodule]
 fn qcs_sdk(py: Python<'_>, m: &PyModule) -> PyResult<()> {
+    pyo3_log::init();
```

Comment:
Should this be `try_init` with a warning message on failure? Seems like this is generally going to succeed, but it's be best to not brick the package when it doesn't for whatever reason.